cli/requirements.txt

```diff
@@ -2,4 +2,5 @@ sphinx>=1.3.1, <2.0
 tox>=2.2, <3.0
 wheel>=0.24.0, <1.0
 pytest>=2.9.1
+teamcity-messages>=1.20
```

Comment:
I don't _think_ so. 

This causes the OS X CLI build which does a `make test-binary` which means it only gets what is in cli/requirements.txt installed to it, which includes pytest but not teamcity-messages which resulted in it not showing nice structured test results.

The top level requirements.txt doesn't have pytest, just tox, and so I think not having it there is good / right (there is nothing that would use teamcity-messages, just tox, and if someone runs tox, that adds the dependency).